include/nn/oe.h

```diff
@@ -43,5 +43,6 @@ void SetUserInactivityDetectionTimeExtended(bool);
 void FinishStartupLogo();
 nn::settings::LanguageCode GetDesiredLanguage();
 void GetDisplayVersion(DisplayVersion*);
+bool TryPopLaunchParameter(void*, u64); // TODO : figure out types for this
```

Comment:
That's not the correct signature. This should be:

    nn::oe::TryPopLaunchParameter(ulong *,void *,ulong)

according to the symbol, which is probably `nn::oe::TryPopLaunchParameter(size_t*,void *,size_t)`